integer.js

```diff
@@ -0,0 +1,27 @@
+const prompt = require('prompt-sync')();
+
+const x = parseInt(prompt('please assign an integer to x: '));
+
+let value_of_x, value_of_y;
+
+if (x > 0) {
+  value_of_x = true
+} else {
+  value_of_x = false
+}
+
+const y = parseInt(prompt('now please assign an integer to y: '));
+
+if (y > 0) {
+  value_of_y = true
+} else {
+  value_of_y = false
+}
+
+if ((value_of_x == true) && (value_of_y == true)) {
```

Comment:
I would suggest you remove the extra pair of brackets and leave the out the equality checks. Since the variables used already return a boolean value, the additional check is not needed. I would suggest the if statement looks like this:
```suggestion
if (value_of_x  && value_of_y) {
```